Allow starting the commit of a journal transaction, without waiting for
it to complete.  This is a performance enhancement for OST IO so that
the journal commit can run concurrently with the file IO.  It isn't
necessary if the client can handle bulk IO recovery (bug 16919).

Index: linux-2.6/fs/jbd/journal.c
===================================================================
--- linux-2.6.orig/fs/jbd/journal.c	2006-07-15 16:13:50.000000000 +0800
+++ linux-2.6/fs/jbd/journal.c	2006-07-15 16:22:04.000000000 +0800
@@ -74,6 +74,7 @@ EXPORT_SYMBOL(journal_abort);
 EXPORT_SYMBOL(journal_errno);
 EXPORT_SYMBOL(journal_ack_err);
 EXPORT_SYMBOL(journal_clear_err);
+EXPORT_SYMBOL(log_start_commit);
 EXPORT_SYMBOL(log_wait_commit);
 EXPORT_SYMBOL(journal_start_commit);
 EXPORT_SYMBOL(journal_force_commit_nested);
